### PR TITLE
Libraries mapping, accept string argument in parser.

### PR DESCRIPTION
**Pull request by caroo7**
Improvements in converter and mappers.

### CODE REVIEW

File: database/src/main/java/config/DatabaseConfiguration.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public LocalContainerEntityManagerFactoryBean entityManagerFactory() {
         properties.put("hibernate.show_sql", environment.getRequiredProperty("hibernate.show_sql"));
         properties.put("hibernate.format_sql", environment.getRequiredProperty("hibernate.format_sql"));
         properties.put("hibernate.hbm2ddl.auto", environment.getRequiredProperty("hibernate.hbm2ddl.auto"));
-        properties.put("hibernate.hbm2ddl.import_files", environment.getRequiredProperty("hibernate.hbm2ddl.import_files"));
+        //properties.put("hibernate.hbm2ddl.import_files", environment.getRequiredProperty("hibernate.hbm2ddl.import_files"));
```
Comment (tdudzik) [inline database/src/main/java/config/DatabaseConfiguration.java:50]: Commented code in a repo? :disappointed:

File: parser/pom.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@
                         </goals>
                         <configuration>
                             <finalName>parser-executable</finalName>
-                            <mainClass>main.main.Main</mainClass>
+                            <mainClass>main.ParserMain</mainClass>
```
Comment (tdudzik) [inline parser/pom.xml:68]: :+1:

**Comment by tdudzik** [comment]
For me it's ok and can be merged.

File: database/src/main/java/config/DatabaseConfiguration.java (hunk 1 above)
Comment (caroo7) [reply database/src/main/java/config/DatabaseConfiguration.java:50]: I left this line intentionally because I don't want to forget about this property.

Comment (LIttleAncientForestKami) [reply database/src/main/java/config/DatabaseConfiguration.java:50]: Change such cases in TODO please. Or mark them as such. Or create a Github issue for that. Not a commented line. :-1:

**Pull request merged**